### ISSUE TITLE
Get-Help does not list all 'About*' topics 

### ISSUE THREAD

**Issue by MaximoTrinidad** [PowerShell/PowerShell#2653]
Steps to reproduce
------------------
PS /home/maxt> Get-Help About* | Select name 

Expected behavior
-----------------
To provide a list of all available 'About_*' topics.

Actual behavior
---------------
Is showing nothing to list.
![gethelpnoaboutlist](https://cloud.githubusercontent.com/assets/19892435/20153899/f2769f08-a692-11e6-96fa-2bdf6a36d777.png)

Environment data
----------------
OS = Ubuntu 16.04.1 LTS Desktop

```powershell
PS /home/maxt> $PSVersionTable                                                  

Name                           Value                                           
----                           -----                                           
PSVersion                      6.0.0-alpha                                     
PSEdition                      Core                                            
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                         
BuildVersion                   3.0.0.0                                         
GitCommitId                    v6.0.0-alpha.12                                 
CLRVersion                                                                     
WSManStackVersion              3.0                                             
PSRemotingProtocolVersion      2.3                                             
SerializationVersion           1.1.0.1

```

**Comment by joeyaiello** [comment]
That's cause we haven't shipped them yet. :)

There's a bunch of problems with about_ topics that need to be addressed by our new publishing pipeline. They'll likely all get fixed at once (or very close together), but I don't have an ETA yet, unfortunately...

**Comment by iSazonov** [comment]
Is the original Issue only on Unix? I cannot reproduce on Windows.

@SteveL-MSFT  If we want fix this in 6.0.0-beta1 we need more details about what @joeyaiello  said. Or will it be an internal solution?

**Comment by gonhidi** [comment]
I have experienced this issue using v6.0.0-alpha.17, both on macOS 10.12.3 Sierra and on Ubuntu 16.04 LTS (using the AppImage): Get-Help about_* lists no topics but Get-Help about_Variables does return the appropriate help page.

**Comment by SteveL-MSFT** [comment]
This doesn't repro on Win10, but does repro on Ubuntu16 with alpha.17 for me.

**Comment by iSazonov** [comment]
@chunqingchen Do we ready to close the Issue?

**Comment by SteveL-MSFT** [comment]
This still repros with #3528.  This one won't make it for beta1.

**Comment by joeyaiello** [comment]
Next step here is to list any missing about_* topics (it should be a short or non-existent list), so we can validate before closing.

**Comment by chunqingchen** [comment]
This is verified being fixed by #3852
